src/generator/resolve_deps.jl

```diff
@@ -116,6 +116,8 @@ function resolve_dependency!(dag::ExprDAG, node::ExprNode{TypedefDefault}, optio
         push!(node.adj, dag.ids[leaf_ty.sym])
     elseif haskey(dag.ids_extra, leaf_ty.sym)
         # pass
+    elseif startswith(spelling(getTypeDeclaration(ty)), "__builtin_")
```

Comment:
Is it possible to solve this problem by using `@add_def` macro?

Reply:
I originally tried `"output_ignorelist" => ["__darwin_va_list"]`, but that only suppresses printing.

I wasn't aware of `@add_def`, but it doesn't do quite what I want either. It creates a new type in the calling module. e.g. ...

```julia
julia> module Foo
       using Clang
       using Clang.Generators
       @add_def bar
       end
Main.Foo

julia> Foo.bar isa DataType
true
```

What I really want is to ignore `__darwin_va_list` completely. 

It looks like I can solve the problem by doing this:
```julia
    Generators.add_definition(:__builtin_va_list => Generators.JuliaUnsupported())
```

I will remove this commit from the PR.